cmd/influx/main.go

```diff
@@ -249,6 +258,22 @@ func (c *CommandLine) SetFormat(cmd string) {
 	}
 }
 
+func (c *CommandLine) dump() {
+	response, err := c.Client.Dump(c.Database)
+	defer response.Close()
+	if err != nil {
+		fmt.Printf("Dump failed. %s\n", err)
+	} else {
+		scanner := bufio.NewScanner(response)
```

Comment:
I think you can do:

```go
_, err := io.Copy(os.Stdout, response)
```

See this example
http://play.golang.org/p/lK2kFDJUw0